functests/3_performance_status/status.go

```diff
@@ -0,0 +1,62 @@
+package __performance_status
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	testutils "github.com/openshift-kni/performance-addon-operators/functests/utils"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/mcps"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/nodes"
+	"github.com/openshift-kni/performance-addon-operators/functests/utils/profiles"
+	v1 "github.com/openshift/custom-resource-status/conditions/v1"
+	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var _ = Describe("Negative testing of performance profile", func() {
+	var workerRTNodes []corev1.Node
+	var err error
+
+	BeforeEach(func() {
+		workerRTNodes, err = nodes.GetByRole(testutils.RoleWorkerCNF)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(workerRTNodes).ToNot(BeEmpty())
+	})
+
+	Context("Performance Addons detailed status", func() {
+		var nodeAnnotationConfig = "machineconfiguration.openshift.io/currentConfig"
+		var currentConfig string
+		nodeLabel := map[string]string{fmt.Sprintf("%s/%s", testutils.LabelRole, testutils.RoleWorkerCNF): ""}
+		annotations := workerRTNodes[0].GetAnnotations()
+		for k, v := range annotations {
+			if k == nodeAnnotationConfig {
+				currentConfig = v
+			}
+		}
+
+		It("[test_id:29673] Machine config pools status tied to Performance Profile", func() {
+
+			currentConfigAnnotation := map[string]string{
+				nodeAnnotationConfig: currentConfig,
+			}
+			updateAnnotation := map[string]string{
+				nodeAnnotationConfig: "",
+			}
+
+			// Empty the value of "machineconfiguration.openshift.io/currentConfig" for node with worker-cnf label
+			By("Applying changes in machineconfiguration currentConfig of CNF worker node")
+			reason := "1 nodes are reporting degraded status on sync"
+			workerRTNodes[0].SetAnnotations(updateAnnotation)
+			// Wait until worker-cnf MCP is in degraded state and get condition reason
+			mcps.WaitForCondition(testutils.RoleWorkerCNF, machineconfigv1.MachineConfigPoolDegraded, corev1.ConditionTrue)
+			conditionReason := mcps.GetConditionReason(testutils.RoleWorkerCNF, machineconfigv1.MachineConfigPoolNodeDegraded)
+			profileMessage := profiles.GetProfileMessage(nodeLabel, v1.ConditionDegraded)
+			// Verify the status reason of performance profile
+			Expect(conditionReason).To(ContainSubstring(reason))
+			Expect(profileMessage).To(ContainSubstring(reason))
```

Comment:
Maybe `Expect(profileMessage).To(ContainSubstring(conditionReason))`?
And please rename `conditionReason` to `mcpConditionReason` to make it easier to see that it is the reason of the MCP's condition.

Reply:
Done. Also, I removed testing mcpConditionReason.